### ISSUE TITLE
[Tool bar] Tooltips in Tooltips and Icon Only stories are not announced by screen readers

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3264]
### 😯 Current Behavior

https://github.com/user-attachments/assets/fd365e6c-2d52-402f-ae37-bb1d0654c76f

In DevTools inside Accessibility tab there is no value provided for `aria-labelledby`:
<img width="304" height="242" alt="Image" src="https://github.com/user-attachments/assets/5fd319a2-c257-4694-9dff-79190330a154" />

https://github.com/user-attachments/assets/72b7161c-37e6-4edb-8f72-81a29d0e0c68

### 🤔 Expected Behavior/possible solution?

Tooltip content should be announced as a name of each button.

https://github.com/user-attachments/assets/e5a12812-e015-4d4e-977b-d16d6a307dbc

<img width="281" height="294" alt="Image" src="https://github.com/user-attachments/assets/88ed6241-444d-4023-933e-0848ce0bf8de" />

### 🖥️ Steps to Reproduce

1. Go to https://storybook.sanomalearning.design/?path=/story/actions-tool-bar--tooltips or https://storybook.sanomalearning.design/?path=/story/actions-tool-bar--icon-only
2. Open DevTools.
3. In Accessibility tab check if buttons with tooltip has `aria-labelledby` with value added 
4. With screen reader turned on go to buttons with tooltips. 
5. Check if tooltip content is announced as button name.

### Were you able to reproduce this in our storybook?

Yes

### If yes, provide the storybook URL

https://storybook.sanomalearning.design/?path=/story/actions-tool-bar--tooltips and https://storybook.sanomalearning.design/?path=/story/actions-tool-bar--icon-only

### ⚖️ Impact

This blocks our release or stops us from using this component

### Framework

Other

### What browsers are you seeing the problem on?

All

### Component version

v0.2.3

### 👤 Your name

Kacper

### 🧢 Your product/team

SLDS

### 🎨 Your Theme(s)

Sanoma Learning
